### PR TITLE
Create map_loption_ne_to_downup

### PR DESCRIPTION
**Pull request by yosinlpet**

### CODE REVIEW

File: public/json/map_loption_ne_to_downup [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+          "type": "basic",
+          "from": {
+            "key_code": "n",
+            "modifiers": {
+              "mandatory": [
+                "left_option"
+              ],
+              "optional": [
+                "any"
+              ]
```
Comment (MuhammedZakir) [inline public/json/map_loption_ne_to_downup:17]: Using `any` in `from.modifiers.optional` will map all `left_option+any_other_modifiers+n/e` shortcut up/down arrow. Is this needed? Personally, I would just use `caps_lock`, but it's upto you.

Comment (yosinlpet) [reply public/json/map_loption_ne_to_downup:17]: oops, I wanted the left_option key only

Comment (MuhammedZakir) [reply public/json/map_loption_ne_to_downup:17]: Then, you can either remove the `optional` modifiers, or add `caps_lock` to it if you want your modification to work when caps lock is enabled.

**Comment by MuhammedZakir** [comment]
Please close the other PR, and push `Update groups.json` commit to `yosinlpet:patch-1`.

If you are editing from the website, go to your `patch-1` branch [1], update `groups.json`, and choose the `Commit directly to the patch-1 branch.` option.

[1] https://github.com/yosinlpet/KE-complex_modifications/tree/patch-1

**Comment by tekezo** [comment]
Thanks!
I merged this PR with:

- Rename `map_loption_ne_to_downup` to `map_loption_ne_to_downup.json`.
- Update groups.json. https://github.com/pqrs-org/KE-complex_modifications/commit/7e8d97a64adcb16f607094121c4c7a9cf280ee4b

**Pull request merged**